bitnami/mean/values.yaml

```diff
@@ -4,7 +4,7 @@
 image:
   registry: docker.io
   repository: bitnami/node
```

Comment:
This is supposed to be a MEAN solution: MongoDB+Express+AngularJS+NodeJS

AFAIK **bitnami/node**  does not include Express nor AngularJS by default. I guess it's necessary to use **bitnami/express** or **bitnami/angular**  but those images don't seem to be maintained...

What do you think @tompizmor ?